integration_tests/src/main/python/conftest.py

```diff
@@ -27,13 +29,28 @@
 ]
 
 _approximate_float_args = None
+_pyspark_monkey_patches = []
 
 def get_float_check():
     if not _approximate_float_args is None:
         return lambda lhs,rhs: lhs == pytest.approx(rhs, **_approximate_float_args)
     else:
         return lambda lhs,rhs: lhs == rhs
 
+def add_timestamp_monkey_patches():
+    global _pyspark_monkey_patches
+    from pyspark.sql.types import TimestampType, TimestampNTZType
```

Comment:
Sadly TimestampNTZ was added to spark/pyspark in 3.3.0 https://issues.apache.org/jira/browse/SPARK-37126 and we still support some older versions of Spark. So we need to make this work for all versions we support.